### PR TITLE
api.md: clarify response behaviour for ACL POST endpoint

### PR DESCRIPTION
**Pull request by chungdaniel**
Signed-off-by: Daniel Chung <daniel@tailscale.com>

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -274,10 +274,11 @@ Etag: "e0b2816b418b3f266309d94426ac7668ab3c1fa87798785bf82f1085cc2f6d9c"
 
 #### `POST /api/v2/domain/:domain/acl` - set ACL for a domain
 
-Sets the ACL for the given domain. HuJSON and JSON are both accepted inputs. An `If-Match` header can be set to avoid missed updates.
+Sets the ACL for the given domain.
+HuJSON and JSON are both accepted inputs.
```
Comment (bradfitz) [inline api.md:278]: Does the request `Content-Type` need to be set either way or at all?

File: api.md [hunk 2]
```diff
@@ -287,7 +288,13 @@ Returns error if using an `If-Match` header and the ETag does not match.
 `Accept` - Sets the return type of the updated ACL. Response is parsed `JSON` if `application/json` is explicitly named, otherwise HuJSON will be returned.
 
 ###### POST Body
-ACL JSON or HuJSON (see https://tailscale.com/kb/1018/acls)
+ACL JSON or HuJSON (see https://tailscale.com/kb/1018/acls for more details).
+`Groups` - Static groups of users which can be used for ACL rules.
```
Comment (bradfitz) [inline api.md:292]: The `POST` body should be a JSON or [HuJSON](https://github.com/tailscale/hujson#hujson---human-json) formatted JSON object. An ACL policy may contain the following top-level properties:

* ...
* ...

See https://tailscale.com/kb/1018/acls for more information on those properties.

**Review by c22wen — APPROVED** [review]
Looks good. Maybe you could even give an example of a failed test response if that would be helpful.

File: api.md (hunk 1 above)
Comment (chungdaniel) [reply api.md:278]: `Content-Type` doesn't need to be set for requests, but the server sets it for the response.

**Review by crawshaw — APPROVED** [review]

**Pull request merged**